src/test/java/tests/LoginTests.java

```diff
@@ -41,4 +42,13 @@ public void tc03_LoginWithInvalidPassword() {
         LogHelper.info("Check the error message");
         Assert.assertEquals(loginPage.getErrorLoginMessage(), "Invalid username or password. Please try again.", "Error login message doesn't display correctly");
     }
+
+    @Test(description = "System shows message when user enters wrong password 4 times")
+    public void tc05_LoginWithEnteringWrongPasswordSeveralTimes(){
+        LogHelper.info("Login with wrong password 4 times ");
+        loginPage.repeatLogin(Constants.USER, DataHelper.getRandomPassword());
```

Comment:
no need random invalid password

Reply:
I've already fixed it. Please help me check it. Thanks so much